presto-spi/src/test/java/com/facebook/presto/spi/resourceGroups/TestResourceGroupId.java

```diff
@@ -23,7 +27,7 @@
     @Test
     public void testBasic()
     {
-        new ResourceGroupId("test.test");
+        new ResourceGroupId("test_test");
```

Comment:
We added a check that each segment should not have a "." in it, which makes sense. The old test case should not be valid because `(new ResourceGroupId("test.test")).toString()` is the same as `(new ResourceGroupId(new ResourceGroupId("test"), "test)).toString()`